code/modules/mob/living/simple_animal/hostile/megafauna/blood_drunk_miner.dm

```diff
@@ -30,7 +30,7 @@ Difficulty: Medium
 	icon = 'icons/mob/alienqueen.dmi'
 	light_color = "#E4C7C5"
 	speak_emote = list("roars")
-	speed = 1
+	speed = 3
```

Comment:
Should probably be included in changelog / PR desc

Reply:
That _decreases_ his speed, right? What an odd change, given that my impression was that you had to facetank him

Reply:
the `speed` var only applies to when players are controlling the mob.

Reply:
So this is just to make adminbussed blood drunks a little less ridiculous? Sounds fair